lib/file-source/src/file_watcher/tests/common.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do we need this file?

Reply:
We sure do not. Just junk I accidentally left in-tree.